### ISSUE TITLE
Electricity load tutorial problem

### ISSUE THREAD

**Issue by matsuobasho** [Nixtla/mlforecast#338]
### What happened + What you expected to happen

Didn't see a specific page for submitting issues about the tutorials.
https://nixtlaverse.nixtla.io/mlforecast/docs/tutorials/electricity_load_forecasting.html

In the Test Evaluation section, `df_train` is still being used instead of `df_last_24_hours`.  Also, it doesn't appear that this section is using models from the train section, rather just creating new models.

### Versions / Dependencies

NA

### Reproduction script

NA

### Issue Severity

Low: It annoys or frustrates me.

**Comment by jmoralez** [comment]
Hey.

> In the Test Evaluation section, df_train is still being used instead of df_last_24_hours

What's the problem with this? `df_train` is used to train and `df_last_24_hours` is the test set, so that's merged with the forecasts.

>  Also, it doesn't appear that this section is using models from the train section, rather just creating new models.

Which train section? The previous section performs cross validation to assess the performance across different subsets of the data, it doesn't train a model with all of it.

**Comment by matsuobasho** [comment]
Missed your quick, response, thanks.

Ok, now I understand why I was confused.  "now we are going to evaluate the performance on the test set" makes it seem like we will be feeding `df_last_24_hours` to a model.
I would suggest changing that sentence to:
"Now we are going to predict for the next 24 hours with the training set and compare the performance to the test set."

As for the next sentence:
> We can use both of them for forecasting the test alongside some prediction intervals. 
Not sure what "both of them" refers to here.

**Comment by jmoralez** [comment]
Feel free to make a PR changing the wording.

> Not sure what "both of them" refers to here.

That connects with the previous section:

> We can se that the model lgbm has top performance in most metrics folowed by the lasso regression. Both models perform way better than the naive.
